app/services/datetime_parser.rb

```diff
@@ -6,34 +6,45 @@ def initialize(date:, time:)
     @raw_time = time.to_s
   end
 
+  def issues
+    @issue_items
```

Comment:
I'd consider renaming this variable to `@issues` and then making the attribute accessor public. That way, you won't need to create a "wrapper" method for it as `attr_reader` does that for you.